addons/trenches/CfgWorldsTextures.hpp

```diff
@@ -0,0 +1,280 @@
+class CfgWorldsTextures {
```

Comment:
Usually I'd say class structures added by ace components should use the prefix, however I'm not sure here since I can see that retaining this one would be backwards compatible with the original addon 🤔

Reply:
I don't think any map actually adds compatibility for this in the map mod, so it wouldn't be a problem to change. Additionally backwards compatibility can be simpler by inheriting the class from the new class.

Reply:
Tag everything custom in a global namespace.

Reply:
Maybe this data should be included into `CfgWorlds` as it's referring to single maps.What about that?